src/components/Navbar.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+function Navbar() {
+  return (
+    <div className="menu">
+
+      <div>
+        <h1>Math Magician</h1>
+      </div>
+      <ul>
+        <li><Link to="/home">Home</Link></li>
```

Comment:
- [ ] Once you leave only one route don't forget to change the path for the link that points to `Home`.

```suggestion
        <li><Link to="/">Home</Link></li>
```